src/main/java/org/folio/links/NoteLinksConstants.java

```diff
@@ -47,6 +47,8 @@ private NoteLinksConstants() {
 
   static final String ORDER_BY_NOTE_TYPE = "ORDER BY type.jsonb->>'name' %s ";
 
+  static final String ORDER_BY_UPDATED_DATE = "ORDER BY data.jsonb->'metadata'->>'updatedDate' %s ";
```

Comment:
Will this order statement sort `updatedDate` like strings or like date objects? I just think if this will lead to problems.

Reply:
I got your point. Thanks Pavel. I will check and reply

Reply:
@psmagin fixed